src/main/java/duke/commands/IncorrectCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.commands;
+
+/**
+ * Implements the incorrect command, which handles invalid user input.
+ * @author Lim Yong Shen, Kevin
+ */
+public class IncorrectCommand extends Command {
+
```

Comment:
Perhaps using exceptions would be a better approach? This makes this class look like a working class part of the functionality.

Reply:
I did consider using an Exception but I made the decision to create this class because I felt like any incorrect inputs could still be encapsulated within their own command. I appreciate the feedback and will look into using custom Exceptions in future.